Detectors/YOLO/yolo.py

```diff
@@ -19,4 +24,14 @@ def detect(self, image):
 
         Returns a list of detections where each item in list contains [x1, y1, x2, y2, conf]
         """
-        pass
\ No newline at end of file
+
+        width = darknet.network_width(self.m)
+        height = darknet.network_height(self.m)
+        darknet_image = darknet.make_image(width, height, 3)
+        image_rgb = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
+        image_resized = cv2.resize(image_rgb, (width, height),
+                                interpolation=cv2.INTER_LINEAR)
+        darknet.copy_image_from_bytes(darknet_image, image_resized.tobytes())
+        detections = darknet.detect_image(self.m, self.class_names, darknet_image)
+        res=darknet.draw_boxes(detections, image, self.colors)
+        return detections, res
```

Comment:
1. Only return `detections` which is required by the tracker to maintain uniformity while using different detection methods.
2. Condition for taking account the offset of the bounding box is not implemented.
3. Add a newline after `return` statement